### ISSUE TITLE
map API implementation with Snap/Mapbox confuses the user

### ISSUE THREAD

**Issue by jpbonjour** [zdmc23/paraguide-public#2]
1. Launch the app
2. Click on disclosure triangle at the bottom of the image for detailed information
3. In Map view interact with the map. 

Outcome: It takes the user to the Snap app in the appStore or loads the Mapbox UI inside of the map view (Snaps 3rd party service for mapping) 

Additional feedback: There is a search box presented inside of the map view - when a locality is entered it only moves that map view. The users expected outcome is to switch Paraguides suggestions to new locality. 

![img_5580](https://user-images.githubusercontent.com/41528698/47966579-e15b0900-e008-11e8-933e-1900a1720cb9.PNG)

**Comment by zdmc23** [comment]
You're right that we don't want to entice users to leave the app, but at the same time, we're fairly limited on data with this MVP, so we wanted to not leave early users stuck with lack of info.  We thought SnapMaps would be fun (though, I don't think it's been all that useful beyond demonstrating potential integration possibilities).  The Web was the fall back in case someone wanted a phone number, or to make a reservation, etc... as we expand our data offering, then I think that button becomes unnecessary.
